Gemfile.lock

```diff
@@ -297,4 +300,4 @@ DEPENDENCIES
   xcpretty-travis-formatter!
 
 BUNDLED WITH
-   2.1.4
+   2.2.5
```

Comment:
Did we mean to change this?

Reply:
I did it intentionally, yes. The reason being that Bundler 2.2.x fixes an issue that may result in confusing changes to the `.bundle/config` file (more details in this Simplenote iOS PR https://github.com/Automattic/simplenote-ios/pull/1059).

Do you think it's a problem? An out-of-scope change on a PR that should be as little as possible given it targets the release branch?

Reply:
I'm good with it! Just wanted to be sure it was intentional!